src/common/types.hpp

```diff
@@ -143,6 +144,12 @@ class Ip6Address
      */
     Ip6Address(const uint8_t (&aAddress)[16]);
 
+    /**
+     * Constructor with an otIp6Address.
+     *
+     */
+    Ip6Address(const otIp6Address &aAddress);
```

Comment:
```suggestion
    explicit Ip6Address(const otIp6Address &aAddress);
```

Reply:
Added 'explicit' but keep the comment for consistency.